TomTom SDK Examples/Examples/UseCase/Offline/Utils/OfflineMapHelper.swift

```diff
@@ -0,0 +1,137 @@
+//  © 2023 TomTom NV. All rights reserved.
+//
+//  This software is the proprietary copyright of TomTom NV and its subsidiaries and may be
+//  used for internal evaluation purposes or commercial use strictly subject to separate
+//  license agreement between you and TomTom NV. If you are the licensee, you are only permitted
+//  to use this software in accordance with the terms of your license agreement. If you are
+//  not the licensee, you are not authorized to use this software in any manner and should
+//  immediately return or destroy it.
+
+import Foundation
+
+// MARK: Offline Map Errors
+
+enum OfflineMapError: LocalizedError {
+    case bundleResourceURLNotExist
+    case documentsDirectoryNotExist
+    case mapFolderDoesNotExist
+    case keystoreFileDoesNotExist
+
+    public var errorDescription: String? {
+        /* YOUR CODE GOES HERE */
+        return "Error! . \(self)"
+    }
+}
+
+// MARK: Offline Map Helper
+
+enum OfflineMapHelper {
+    /// Copy the 'Map' folder (all folders and files recursively inside) from the Bundle to the device
+    static func copyFolders(completion: (_ success: Bool, _ error: Error?) -> Void) {
+        let fileManager = FileManager.default
+        
+        guard let pathFromBundle = Bundle.main.resourceURL?.appendingPathComponent(OfflineConfig.mapFolderName).path else {
+            completion(false, OfflineMapError.bundleResourceURLNotExist)
+            return
+        }
+        
+        guard let docsURL = fileManager.urls(for: .documentDirectory, in: .userDomainMask).first else {
+            completion(false, OfflineMapError.documentsDirectoryNotExist)
+            return
+        }
+        let pathDestDocs = docsURL.appendingPathComponent(OfflineConfig.mapFolderName).path
+
+        if fileManager.fileExists(atPath: pathDestDocs) {
+            /// It has already been copied previously
+            /* YOUR CODE GOES HERE */
+            completion(true, nil)
+        } else {
+            /// Copy directory content recursively
+            do {
+                let files = try fileManager.contentsOfDirectory(atPath: pathFromBundle)
+                try? fileManager.copyItem(atPath: pathFromBundle, toPath: pathDestDocs)
+
+                for fileName in files {
+                    try? fileManager.copyItem(atPath: "\(pathFromBundle)/\(fileName)", toPath: "\(pathDestDocs)/\(fileName)")
```

Comment:
https://github.com/tomtom-international/tomtom-navigation-ios-examples/pull/3/files#r1180605217

Reply:
It's fixed!